docs/technical-details/self-hosted/database.md

```diff
@@ -27,6 +27,26 @@ postgresql:
     enabled: false
 ```
 
+By default, the Fairwinds Insights Helm chart will install and configure the CloudNativePG (CNPG) operator for you. This is controlled by the following configuration in `values.yaml`:
+
+```yaml
+postgresql:
+  operator:
+    install: true # Set to true to install the CNPG operator
+    webhook:
+      mutating:
+        create: true
+      validating:
+        create: true
+    crds:
+      create: true # Manages CRDs for CNPG
+```
+
+**Note:**  
+If you already have the CNPG operator installed and configured in your cluster, you should set these options to `false` to prevent the chart from managing the operator or its webhooks/CRDs.
```

Comment:
```suggestion
If you already have the CNPG operator installed and configured in your cluster, you should set the `postgresql.operator.install` option to `false` to prevent the chart from managing the operator or its webhooks/CRDs.
```